external/kafka/src/test/scala/org/apache/spark/streaming/kafka/DirectKafkaStreamSuite.scala

```diff
@@ -381,3 +447,20 @@ object DirectKafkaStreamSuite {
     }
   }
 }
+
+private[streaming] class ConstantEstimator(rates: Double*) extends RateEstimator {
```

Comment:
@tdas I tried to reuse that but that is in different project. Is the test files from streaming project shared with external projects?